mock/api.js

```diff
@@ -6,41 +6,24 @@ const DEFAULT_SIZE = 100;
 export const mockIndices = new Array(DEFAULT_SIZE).fill().map(() => moment().format('YYYY-MM'));
 
 export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-  controller: casual.word + index,
-  key: casual.word + index,
-  last_modified_string: moment.utc() + index,
+  controller: `${casual.word}.${casual.word}.com`,
```

Comment:
That's an interesting way to do it.

I didn't think to mention this before; however to be really consistent the value returned here should match the `hostname` parameter in your `/datasets/list` mock; because this is the controller name the dashboard will be requesting there.

If the dashboard cared about that consistency, this would be an issue.

Reply:
Fixed !